edx_exams/apps/api/models.py

```diff
@@ -1,4 +1,108 @@
-# Models that can be shared across multiple versions of the API
-# should be created here. As the API evolves, models may become more
-# specific to a particular version of the API. In this case, the models
-# in question should be moved to versioned sub-package.
+from model_utils.models import TimeStampedModel
```

Comment:
it feels weird to have these in the API directory

I would expect items in the API vs the core to be specifically tailored to API use - for example combining attributes from three DB models into a single domain concept for the front end, or providing a simplified view of a database model, or whatever

Reply:
or maybe because core has the user model which connects back to the rest of edx these models deserve their own exam app. or the user model should be evicted into a separate user app

Reply:
Looking at other apps, the models are just all grouped within core/models.py. I'll do the same here.